package.json

```diff
@@ -50,6 +50,11 @@
     "configuration": {
       "title": "Kotlin",
       "properties": {
+        "kotlin.java.opts": {
+          "type": "string",
+          "default": "",
+          "description": "Custom options using JAVA_OPTS for the language server and debug adapter."
```

Comment:
Since we do not support the debug adapter, the description should be adjusted accordingly.

Reply:
U are right, this PR comes from <https://github.com/fwcd/vscode-kotlin/pull/121>，I copied it's desciption.Now I fixed it, you can review it again